File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/Administrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using gov.va.medora.mdo.domain.sm.enums;

namespace gov.va.medora.mdo.domain.sm
{
    public class Administrator : gov.va.medora.mdo.domain.sm.User
    {
        // national administrator?
        public bool National { get; set; }

        // list of facilities that this administrator can manipulate
        public List<Facility> Facilities { get; set; }

        // list of VISNs that this administrator can manipulate
        private List<Facility> Visns { get; set; }

        public Administrator()
        {
            //super();
            //UserType = UserTypeEnum.ADMINISTRATOR;
            //ParticipantType = null;
        }

    }
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/Administrator.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;
import java.util.Collection;

import gov.va.med.mhv.sm.enumeration.UserTypeEnum;

public class Administrator extends User implements Serializable{

	private static final long serialVersionUID = -4604433718982828729L;

	// national administrator?
	private boolean national;
	
	// list of facilities that this administrator can manipulate
	private Collection<Facility> facilities;
	
	// list of VISNs that this administrator can manipulate
	private Collection<Facility> visns;
	
	public Administrator(){
		super();
		this.userType = UserTypeEnum.ADMINISTRATOR;
		this.participantType = null;
	}


	public boolean isNational() {
		return national;
	}
	public void setNational(boolean national) {
		this.national = national;
	}
	public Collection<Facility> getFacilities() {
		return facilities;
	}
	public void setFacilities(Collection<Facility> facilities) {
		this.facilities = facilities;
	}
	public Collection<Facility> getVisns() {
		return visns;
	}
	public void setVisns(Collection<Facility> visns) {
		this.visns = visns;
	}
}
